lib/wallaroo/core/initialization/local_topology.pony

```diff
@@ -1626,6 +1684,18 @@ actor LocalTopologyInitializer is LayoutInitializer
   be source_ids_query(conn: TCPConnection) =>
     _router_registry.source_ids_query(conn)
 
+  be state_entity_query(conn: TCPConnection) =>
```

Comment:
We had initially thought it was clearer to call this `state_entity` but now that we've resolved the ambiguity around the term "partition" in the code, let's rename this `state_partitions_query` and we can open an issue to update it in the utilities (and in the tests that use the utilities).

Reply:
see #2268